docker-compose.yaml

```diff
@@ -209,12 +210,24 @@ services:
       - workspace:${WORKSPACE_ROOT}
     networks:
       - airbyte_internal
+  #airbyte-connector-builder: #FIXME: Uncomment this block when enabling airbyte-connector-builder
```

Comment:
@girarda I wonder if we should call this `airbyte-connector-builder-server`, to make it clear that this is only the backing server for the connector builder, and not the connector builder webapp?

Reply:
@lmossman do you think we should rename the module too?